packages/web/package.json

```diff
@@ -5,10 +5,16 @@
   "author": "Audius",
   "version": "1.1.10",
   "private": true,
+  "browser": {
```

Comment:
I'm not 100% sure this will work. For the sdk rollup we used an "ignore" plugin for rollup that replaces given modules with an empty module. And with metro in react native we mocked modules with empty manually:

https://github.com/AudiusProject/audius-client/blob/main/packages/mobile/metro.config.js#L58

Reply:
yeah i think youre right, it wasnt helping anyway, just throwing things at the wall! yeah i did end up just doing that for wepack

Reply:
I believe the browser field is only relevant when the package is a dependency of something else. So if we were importing `audius-client` somewhere and bundling that